SECP-service/SECP.yml

```diff
@@ -2,16 +2,16 @@ sampleProperty: hello!
 
 # Database settings.
 database:
-  driverClass: org.apache.derby.jdbc.EmbeddedDriver
-  url: jdbc:derby:/tmp/mydb;create=true
-  user: admin
-  password: admin
+  driverClass: com.mysql.jdbc.Driver
+  url: jdbc:mysql://localhost:3306/secp?createDatabaseIfNotExist=true
```

Comment:
will this port work for everyone?

Reply:
I was having trouble yesterday with the url for the server, ali was helping me with it but this looks like it will fix it for the future 👍

Reply:
Yes it would work for everyone. When you download Mysql, it automatically creates a server for you at 3306.